_pages/ato/ssp.md

```diff
@@ -94,8 +100,3 @@ user-types:
 controls:
 - <links to your security documentation>
 ```
-
-## Examples
-
-* [ATF eRegs](https://github.com/18F/atf-eregs/blob/master/system-security-plan.yml)
-* [Every Kid In a Park (EKIP) API](https://github.com/18F/ekip-api/blob/master/system-security-plan.yml)
```

Comment:
I thought these files were related to compliance masonry, and therefore that we'd have more contemporary examples soon. You corrected me above, so I'll put them back in.